riff-raff.yaml

```diff
@@ -51,7 +59,11 @@ deployments:
   grid-enr-cloudformation:
     stacks: [media-service]
     template: cloudformation
-    app: grid-es-node-rotation
+    parameters:
```

Comment:
Why doesn't grid have a different `cloudFormationStackName` prefix, when `ophan` and `content-api` do?

Reply:
No strong reason. We've just not been consistent in the deployment of this infrastructure.

Reply:
... and fixing this now is too much of a hassle as we'd have to recreate stacks, fair enough.

Reply:
I think recreating the stacks wouldn't be too troublesome, as the lambdas are configured to run once a week - we'd just have to co-ordinate a little with teams.

This work was part of an effort to CDK this service, however it's since lowered in priority.